pilot/test/integration/testdata/rule-fault-injection.yaml.tmpl

```diff
@@ -1,27 +1,26 @@
 apiVersion: config.istio.io/v1alpha2
-kind: RouteRule
+kind: V1alpha2RouteRule
 metadata:
-  name: fault-route
```

Comment:
why remove the names? They are supposed to be useful for the decorators and help people understand. Unless you want each route to override other.

Reply:
Oh, never mind... ignore the comment.